distributor-api-v1/operations.md

```diff
@@ -517,10 +506,36 @@ If the hotel does not use any payment gateway, the value is null. If it does, th
 | `Description` | [Localized text](./operations.md#localized-text) | required | Description of the product localized into all supported languages. |
 | `ImageId` | string | optional | Unique identifier of the product’s image. |
 | `IncludedByDefault` | boolean | required | Indicates whether the product should be added to order by default. |
-| `Amounts` | array of [Amount](./operations.md#multi-currency-amount) | required | Array of amounts of the product. Only currencies that the property accepts are listed. |
+| `Pricing` | [Pricing coproduct](./operations.md#pricing-coproduct) | required | Object defining the pricing method and price values. |
 | `Charging` | string [Product charging](./operations.md#product-charging) | required | Charging of the product. |
 | `Posting` | string [Product posting](./operations.md#product-posting) | required | Posting of the product. |
 
+#### Pricing coproduct
+
+| Property | Type | Contract | Description |
+| :--- | :--- | :--- | :--- |
+| `Discriminator` | string [Pricing data discriminator](./operations.md#pricing-discriminator) | required | Determines type of value. |
+| `Value` | [Multi-currency amount](./operations.md#multi-currency-amount) / [Relative price data value](./operations.md#relative-price-data-value) | required | Structure of object depends on the [Pricing data discriminator](./operations.md#pricing-discriminator) |
+
+##### Pricing data discriminator
+
+* `Absolute` - Data specific to absolutely priced product are represented by [Multi-currency amount](./operations.md#multi-currency-amount).
+* `Relative` - Data specific to relatively priced product are represented by [Relative price data value](./operations.md#relative-price-data-value).
+
+##### Relative price data value
+
+| Property | Type | Contract | Description |
+| :--- | :--- | :--- | :--- |
+| `ProductIds` | array of string | required | Product ids used to calculate the price of the relatively priced product. |
```

Comment:
Nit: Maybe `array of strings`? `Array of string` sounds weird to me.

Reply:
Nit: I would go with product `IDs` instead of `ids`.

Reply:
I am following the same standard as Connector here. So vice versa, I did update the occurance of `array of strings` to `array of string` everywhere to clean it and use only one version.

`ids` renamed to `Unique identifiers` again, globally in the entire file.